akka-projection-cassandra/src/main/scala/akka/projection/cassandra/CassandraOffsetStore.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+package akka.projection.cassandra
+
+import scala.concurrent.Future
+
+import akka.Done
+import akka.annotation.ApiMayChange
+import akka.annotation.DoNotInherit
+import akka.projection.ProjectionId
+
+@ApiMayChange
+@DoNotInherit
+trait CassandraOffsetStore {
+
+  // FIXME compared to slick.OffsetStore this has the Cassandra prefix in the name?
```

Comment:
`OffsetStore` :+1:

Reply:
Not sure what @seglo is +1 to :-) but I prefer `CassandraOffsetStore` over `OffsetStore`.

I think users may eventually mix a single `Source` with multiple, specialized projections, each using a different `OffsetStore`implementation. In that case, making sure the correct instance is imported without having to navigate to the `import`'s is nice. I can't count how many times I imported the wrong `CassandraSession` or the wrong `Offset` types because both the Akka and the Lagom variants were in scope on a Lagom app.

Reply:
Let me open a separate PR about this. I think it can be internal.

Reply:
oh, now I see this comment. Yes, should be internal. Not directly useful for users (see my comment [below](https://github.com/akka/akka-projection/pull/54#issuecomment-617631197)).

Reply:
I'll fix this after merging this PR

Reply:
FTR I was for calling it simply `OffsetStore`, but if it's internal it matters less :)